third_party/upb/BUILD

```diff
@@ -0,0 +1,410 @@
+load(
```

Comment:
Remove this file?

Reply:
This is now okay to keep this since `third_party/upb` is added to `.bazelignore` which instructs bazel to ignore, which is equivalent to what I expected when removing it.